pkg/sql/opt/xform/testdata/coster/virtual-scan

```diff
@@ -4,11 +4,11 @@ SELECT * FROM information_schema.schemata WHERE SCHEMA_NAME='public'
 select
```

Comment:
i think we'd also want a test (not sure if here or somewhere else) that verifies that we do use the virtual lookup join when the join condition only selects one row from the virtual index.